server/server.go

```diff
@@ -305,6 +313,7 @@ func (s *Server) serviceAccountHandler(logger *log.Entry, w http.ResponseWriter,
 				write(logger, w, fmt.Sprintf("%s\n", path))
```

Comment:
If the code ends up in this branch (call to `.../v1/instance/service-accounts/${account_name}/`, we will not set the `Metadata-Flavor` header. Google's direct API does. It would be good to add it here too for consistency:
```
curl -v http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/ -H "Metadata-Flavor: Google"
*   Trying 169.254.169.254...
* TCP_NODELAY set
* Connected to metadata.google.internal (169.254.169.254) port 80 (#0)
> GET /computeMetadata/v1/instance/service-accounts/default/ HTTP/1.1
> Host: metadata.google.internal
> User-Agent: curl/7.52.1
> Accept: */*
> Metadata-Flavor: Google
>
< HTTP/1.1 200 OK
< Metadata-Flavor: Google
< Content-Type: application/text
< ETag: 1d3dba37416a23dc
< Date: Tue, 09 Jul 2019 17:36:03 GMT
< Server: Metadata Server for VM
< Content-Length: 36
< X-XSS-Protection: 0
< X-Frame-Options: SAMEORIGIN
<
aliases
email
identity
scopes
token
* Curl_http_done: called premature == 0
* Connection #0 to host metadata.google.internal left intact
```